packages/tooltip/src/Tooltip.tsx

```diff
@@ -10,11 +10,27 @@ import Popover, {
 import { useEventListener, useEscapeKey } from '@leafygreen-ui/hooks';
 import { css, cx } from '@leafygreen-ui/emotion';
 import { uiColors } from '@leafygreen-ui/palette';
-import { HTMLElementProps } from '@leafygreen-ui/lib';
+import { HTMLElementProps, typeIs } from '@leafygreen-ui/lib';
 import { transparentize } from 'polished';
 import debounce from 'lodash/debounce';
 import { trianglePosition } from './tooltipUtils';
 
+/**
+ * Converts any type to an array of that type if it isn't already an array,
+ * or an empty array for nullish values.
+ * */
+function toArray<T>(item: T): Array<any> | Array<T> {
```

Comment:
This doesn't exactly behave as described. `toArray(42)` will return `Array<any> | Array<number>`. You probably want an overload. [Example](https://www.typescriptlang.org/play/#code/KYDwDg9gTgLgBAMwK4DsDGMCWEVwIZRR4CeAFJjMALYD8AXPisQJQMXVyYDOcAgoSQA8qANYoIAdxQA+OAG8AUHDhRgMJFFzsqnFFxh50wCAj4DiAbgUBfBXeTosOODAj8iZbQxRIANr7gAHzhUABNgBEwUYFDWOABtAF0rBwxsXFd3IQAVaXJKKgYs4kFcuOLS6RTUNOdM80r86gZs8obc6sd0lzd2vK84VvklTlMmnQBeCbgff2Zh5WVVdU0E5JHbEcwxgg9x5nnFRZU1DS0Cq2VNpdPV+O11zYU0HH18OGn6vYAWACZmKwvPTwABGHx6xVI8T+iQBz1e8DQ4K+JFIs18cKBb1CyN6ezCESiMQBQA)

Reply:
You're right – I'll give an overload here a shot.

Reply:
Could this be solved with `Array.from()` ?